.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+version: 2.1
+
+orbs:
+  browser-tools: circleci/browser-tools@1.1
+
+  # Always take the latest version of the orb, this allows us to
+  # run specs against Solidus supported versions only without the need
+  # to change this configuration every time a Solidus version is released
+  # or goes EOL.
+  solidusio_extensions: solidusio/extensions@volatile
+
+commands:
+  libvips:
+    steps:
+      - run:
+          name: Install libvips
+          command: sudo apt-get install -y libvips
+
+jobs:
+  run-specs-with-postgres:
+    executor: solidusio_extensions/postgres
+    steps:
+      - checkout
+      - browser-tools/install-browser-tools
```

Comment:
I've done the same in other extensions, but recently I realized that we can be more precise here and only install Chrome:

https://circleci.com/developer/orbs/orb/circleci/browser-tools#usage-install_chrome

Reply:
Okay. I'll limit it to install only Chrome. Todo.